tests/integration/test_elasticsearch.py

```diff
@@ -54,6 +54,27 @@ def test_domain_es_version(self):
         status = es_client.describe_elasticsearch_domain(DomainName=domain_name)['DomainStatus']
         self.assertEqual(status['ElasticsearchVersion'], '6.8')
 
+    def test_create_indexes_and_domains(self):
+        indexes = ['index1', 'index2']
+        for index_name in indexes:
+            index_path = '{}/{}'.format(self.es_url, index_name)
+            requests.put(index_path, headers=COMMON_HEADERS)
+            req = requests.get('http://localhost:4571' + '/_cat/indices/' + index_name + '?format=json&pretty')
```

Comment:
nitpick: Let's better use `config.PORT_ELASTICSEARCH` here, instead of hardcoding port `4571`